lib/solarwinds_apm.rb

```diff
@@ -7,10 +7,12 @@
 begin
   require 'solarwinds_apm/logger'
   require 'solarwinds_apm/version'
+  require 'opentelemetry-api'
   if ENV.fetch('SW_APM_ENABLED', 'true') == 'false'
     SolarWindsAPM.logger.info '==================================================================='
     SolarWindsAPM.logger.info 'SW_APM_ENABLED environment variable detected and was set to false. SolarWindsAPM disabled'
     SolarWindsAPM.logger.info '==================================================================='
+    require 'solarwinds_apm/noop'
```

Comment:
Just a thought--instead of requiring `solarwinds_apm/noop` in multiple if/else cases, would it work to require at top similar to `opentelemetry-api`, then the actual implementations would override the noop implementation?

Reply:
Yes, it can be done like opentelemetry-api, just updated it